superset-frontend/src/dataMask/actions.ts

```diff
@@ -0,0 +1,62 @@
+/**
```

Comment:
I see the reasoning behind having `dataMask` on the root level, as they will be used in viz, filters, dashboard and explore, but I still feel `superset-frontend/src/dashboard/dataMask` might be more appropriate, as that's where these masks are primarily used.

Reply:
I think it's the same reason why we put `chart` in the root, it's parallel to chart I think... Also this feature have enough functionality to have independent place in root :)